app/javascript/components/partnering_organizations/PartneringOrgSubscription.js

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+
+// @props po_subscriptionsjk dont need this prop
+// @props partnering organization
+
+class PartneringOrgSubscriptions extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      subscriptions = [],
+    };
+  }
+
+  render() {
+    let subscription_list;
+    subscription_list = this.state.subscriptions.map((subscription) => {
```

Comment:
Yah, so instead of doing an API request to get the subscriptions, we're just gonna pass it through props so this should be `this.props.subscriptions.map(...`